packages/client/src/Client.ts

```diff
@@ -71,6 +85,34 @@ export interface Client extends SharedClient<MinimalClient> {
       : Q extends MinQueryDef<any, any, any> ? QueryMetadata
       : never
   >;
+
+  /**
+   * WARNING. THIS METHOD IS EXPERIMENTAL AND NOT SUPPORTED YET.
+   *
+   * It may change at any time and does not follow semantic versioning. Use at your own risk.
+   *
+   *  @alpha
+   */
+  readonly [__EXPERIMENTAL__NOT_SUPPORTED_YET__preexistingObjectSet]: <
```

Comment:
This pre-existing objectset thing is new right? What use case are we adding this for?

Reply:
Its not new, we had it before. Its for people who have a rid for an object set they want to start with.

Reply:
Oh ok sounds good then